next.json

```diff
@@ -701,6 +701,14 @@
             {"value": 1, "name":"linear"}
         ]
     },
+    "address mode": {
```

Comment:
note: in WebGPU we would probably have a longer name like TextureAddressMode or SamplerAddressMode.

Reply:
Could do, but shouldn't FilterMode be TextureFilterMode, then?

Reply:
Or SamplerFilterMode, yeah let's not do that in NXT.